### PR TITLE
Updating integrational test

### PR DESCRIPTION
**Pull request by err508**
Implements tests channel_close and update_fee events to be tested.

### CODE REVIEW

**Comment by LefterisJP** [comment]
one note: It's `integration` tests. Please rename anything you have from `integrational` to `integration` if there is anything in the code (outside of the PR as I checked the PR  itself already). If not disregard.

PR/branch name does not matter.

File: pathfinder/tests/test_blockchain_integration.py [hunk 1]
```diff
@@ -84,7 +85,47 @@ def test_pfs_with_mocked_client(
...
+    ) in channel_descriptions_case_1:
+        fee_info = clients[p2_index].get_fee_info(
+                clients[p1_index].address,
+                nonce=5,
+                percentage_fee=0.11235813,
+                chain_id=2
+        )
+        pathfinding_service.transport.transmit_data(fee_info.serialize_full())
+        gevent.sleep(0)
+        assert token_network.G[clients[p2_index].address][clients[
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: done.

File: pathfinder/tests/test_blockchain_integration.py [hunk 2]
```diff
@@ -84,7 +85,47 @@ def test_pfs_with_mocked_client(
...
+        p1_deposit,
+        p1_transferred_amount,
+        p1_fee,
+        p2_index,
+        p2_deposit,
+        p2_transferred_amount,
+        p2_fee
+    ) in channel_descriptions_case_1:
+        fee_info = clients[p2_index].get_fee_info(
+                clients[p1_index].address,
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: done.

File: pathfinder/tests/test_blockchain_integration.py [hunk 3]
```diff
@@ -84,7 +85,47 @@ def test_pfs_with_mocked_client(
...
+                nonce=1,
+                percentage_fee=0.123123,
+                chain_id=2
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: done.

File: pathfinder/tests/test_blockchain_integration.py [hunk 4]
```diff
@@ -84,7 +85,47 @@ def test_pfs_with_mocked_client(
...
+        p1_deposit,
+        p1_transferred_amount,
+        p1_fee,
+        p2_index,
+        p2_deposit,
+        p2_transferred_amount,
+        p2_fee
+    ) in channel_descriptions_case_1:
+        fee_info = clients[p2_index].get_fee_info(
+                clients[p1_index].address,
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: changed.

File: pathfinder/tests/test_blockchain_integration.py [hunk 5]
```diff
@@ -84,7 +85,47 @@ def test_pfs_with_mocked_client(
...
+                nonce=5,
+                percentage_fee=0.11235813,
+                chain_id=2
+        )
+        pathfinding_service.transport.transmit_data(fee_info.serialize_full())
+        gevent.sleep(0)
+        assert token_network.G[clients[p2_index].address][clients[
+            p1_index].address]['view'].percentage_fee == 0.11235813
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: done.

File: pathfinder/tests/test_blockchain_integration.py [hunk 6]
```diff
@@ -96,6 +137,13 @@ def test_pfs_with_mocked_client(
         p2_transferred_amount,
         p2_fee
     ) in enumerate(channel_descriptions_case_1):
-        pass
+        balance_proof = clients[p2_index].get_balance_proof(
+            clients[p1_index].address,
+            chain_id=2
+        )
+        clients[p2_index].close_channel(clients[p1_index].address, balance_proof)
+        ethereum_tester.mine_blocks(1)
```
[the remark this answers is not in the document]
Comment (err508) [reply pathfinder/tests/test_blockchain_integration.py]: Works outside, changed.

**Pull request merged**